# Hold information about timing.
Header header

# Name of ego noise source
string name

# Type of noise source
# 0 = FAN
# 1 = MOTOR
uint8 type

# Current load
# RPM for FAN
uint16 load